homeassistant/components/airzone/strings.json

```diff
@@ -66,6 +66,11 @@
       "rssi": {
         "name": "RSSI"
       }
+    },
+    "switch": {
+      "on": {
+        "name": "[%key:common::state::on%]"
+      }
```

Comment:
A switch named On doesn't sound logical to me imo. I'd rather opt for using `name=None` (causing the switch to be seen as the main feature of the device, so in this case it would be names `switch.{device_name}` and it would be like "turn {zone} on or off" (which sounds logical, but I think you already have a climate entity per zone))

Another option would be calling the switch something like Running, but it sounds meh.

Also, climate entities support TURN_OFF and TURN_ON actions now. The default behaviour is that it will use HVAC mode to turn on. But maybe it's nice to overwrite the turn on and off actions to make this what you need

Reply:
> A switch named On doesn't sound logical to me imo. I'd rather opt for using `name=None` (causing the switch to be seen as the main feature of the device, so in this case it would be names `switch.{device_name}` and it would be like "turn {zone} on or off" (which sounds logical, but I think you already have a climate entity per zone))

Done in bb564fb4c4

> Also, climate entities support TURN_OFF and TURN_ON actions now. The default behaviour is that it will use HVAC mode to turn on. But maybe it's nice to overwrite the turn on and off actions to make this what you need

This is my attempt to add independent switches and HVAC mode selects (in a future PR) for changing the status on master and slave zones:
https://community.home-assistant.io/t/configuration-and-use-of-airzone-integration/581937/8
https://github.com/Noltari/aioairzone-cloud/issues/6
Because the following discussion has stalled: https://github.com/home-assistant/architecture/discussions/998

Reply:
That has not stalled, in fact, that has been implemented as far as I'm aware

Reply:
> That has not stalled, in fact, that has been implemented as far as I'm aware

Nope, the only thing that has been implemented is the on/off feature flags:
https://github.com/home-assistant/architecture/discussions/982

However, as long as there's an OFF HVAC mode, I won't consider that discussion as implemented:
https://github.com/home-assistant/core/blob/7c8e00e5e062ced0820ab9f777b23e531f668e47/homeassistant/components/climate/const.py#L17-L18